scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/hvqfn.yaml

```diff
@@ -107,3 +107,59 @@ HVQFN-24-1EP_4x4mm_P0.5mm_EP2.6x2.6mm:
   #pad_length_addition: 0.5
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
+
+HVQFN-40-1EP_6x6mm_P0.5mm_EP4.25x4.25mm:
+  device_type: 'HVQFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.nxp.com/docs/en/package-information/SOT618-1.pdf'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    minimum: 5.9
+    nominal: 6.0
+    maximum: 6.1
+  body_size_y:
+    minimum: 5.9
+    nominal: 6.0
+    maximum: 6.1
+  overall_height:
+    minimum: 0.8
+    nominal: 0.85
+    maximum: 1
+  lead_width:
+    minimum: 0.18
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
+
+  EP_size_x:
+    minimum: 3.95
+    maximum: 4.25
+  EP_size_y:
+    minimum: 3.95
+    maximum: 4.25
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [3, 3]
+  #EP_size_limit_x: 2.7  #for relatively large EP pads (increase clearance)
+  #EP_size_limit_y: 2.7  #for relatively large EP pads (increase clearance)
```

Comment:
The recommended footprint has way less paste than the generated pads:
![image](https://user-images.githubusercontent.com/17516572/90989014-9cb2b000-e597-11ea-81b4-9cb97f181043.png)
![image](https://user-images.githubusercontent.com/17516572/90989028-beac3280-e597-11ea-8825-3bc9da2727a8.png)

TBH, I am not sure what to do here. @poeschlr can you help?